apps/frontend/src/components/fap/FapPage.tsx

```diff
@@ -51,6 +53,10 @@ const FapPage = () => {
         />
       ),
     },
+    {
+      name: 'Documents',
+      element: <FapFilesView data={fap} />,
```

Comment:
As a UserOfficer I believe that showing the upload component is sufficient. What do you think?

```suggestion
      element: isUserOfficer ? (
          <FapFiles
            data={fap}
            onFapUpdate={(newFap: Fap): void => setFap(newFap)}
          />
        ) : (
          <FapFilesView data={fap} />
        ),
```

Reply:
I left it so the user officer could see both so that they are able to see what the users of the system will see and where the documents "end up" and have a better understanding of where the documents are "going". I'm happy to remove this from their view if we think it is unnecessary.